tsconfig.json

```diff
@@ -52,6 +52,7 @@
     "lighthouse-core/test/gather/gatherers/http-redirect-test.js",
     "lighthouse-core/test/gather/gatherers/js-usage-test.js",
     "lighthouse-core/test/gather/gatherers/link-elements-test.js",
+    "lighthouse-core/test/gather/gatherers/main-document-content-test.js",
```

Comment:
Type checking would be a pain, and we probably won't need this test once `afterPass` is deprecated.

Reply:
nooooo, but you've been so great about all the tests so far, and this is a nice opportunity to expand! :D